jumpscale/sals/nginx/templates/location_static.conf

```diff
@@ -1,6 +1,6 @@
-    alias  {{ location.path_location }}{{ "/" if not location.path_location.endswith("/") }};
```

Comment:
make sure in threebot that we do rstrip("/") for `path_location` and add / here at the end